__tests__/app.test.js

```diff
@@ -0,0 +1,46 @@
+const app = require ('../app.js');
+const request = require('supertest');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed.js');
+const data = require('../db/data/test-data');
+const endpoints = require('../endpoints.json');
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('/api', () => {
+    test('GET: 200 - responds with an object detailing all available endpoints', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.endpoints).toEqual(endpoints)
+        })
+    })
+})
+
+describe('/api/topics', () => {
+    test('GET 200 - Responds with an array of objects which has two properties slug and decscription', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.topics.length).toBeGreaterThan(0);
```

Comment:
You can be more specific here. As you know the length is 3 you can use toHaveLength(3)